### PR TITLE
fix for azure control vault access

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by thomast1906 — APPROVED** [review]

**Comment by thomast1906** [comment]
@timja fyi

File: 03-interpolated-defaults.tf [hunk 1]
```diff
@@ -7,6 +7,7 @@ data "azurerm_role_definition" "builtin_role_definition" {
 
 locals {
   slug_location = lower(replace(var.location, " ", "."))
+  environment = (var.environment == "perftest") ? "test" : "${var.environment}"
```
Comment (timja) [inline 03-interpolated-defaults.tf:10]: you don't need brackets
```suggestion
  environment = var.environment == "perftest" ? "test" : "${var.environment}"
```

**Review by timja — COMMENTED** [review]
Wouldn't it be better to allow an override variable to be passed in? rather than having the module know about this hack?

**Review by timja — APPROVED** [review]

**Comment by thomast1906** [comment]
@timja change made as per your reference

**Comment by timja** [comment]
I approved it

**Pull request merged**